internal/core/command/types.go

```diff
@@ -39,12 +49,6 @@ type serviceCommand struct {
 
 // Execute sends the command to the command service
 func (sc serviceCommand) Execute() (string, int, error) {
-	if sc.AdminState == contract.Locked {
-		LoggingClient.Error(sc.Name + " is in admin locked state")
-
-		return "", DefaultErrorCode, ErrDeviceLocked{}
-	}
-
 	LoggingClient.Info("Issuing" + sc.Request.Method + " command to: " + sc.Request.URL.String())
```

Comment:
Minor thing -- this should probably be `Debug` instead of `Info`. Otherwise the log is going to fill up.